update iqama screen icon and title

ismaelAbogabal opened the pull request:
# Request 
- update the iqama screen title, center icon 

# Screenshot 
![image](https://github.com/mawaqit/android-tv-app/assets/63953869/31715ff5-84ac-445e-a6a3-e0ee2085f8ed)

# Note 
- the phone face used on the screenshot is much better to my eyes than the default font by Flutter if you agree with me tell me the font face name and I will add it to be the default font face on the app

ibrahim-zehhaf-mawaqit commented on lib/src/pages/home/sub_screens/IqamaSubScreen.dart:
```diff
@@ -49,35 +49,48 @@ class _IqamaSubScreenState extends State<IqamaSubScreen> {
-              ),
-            ).animate().slide(begin: Offset(0, -1)).fade().addRepaintBoundary(),
-          ),
+        // Expanded(
+        //   child: FittedBox(
+        //     fit: BoxFit.scaleDown,
+        //     child: Container(
+        //       height: 30.vh,
+        //       alignment: Alignment.center,
+        //       padding: EdgeInsets.symmetric(horizontal: 4.vw),
+        //       decoration: BoxDecoration(
+        //         borderRadius: BorderRadius.circular(15),
+        //         gradient: LinearGradient(
+        //           begin: Alignment(0, 0),
+        //           end: Alignment(0, 1),
+        //           colors: [
+        //             theme.primaryColor,
+        //             theme.primaryColor.withOpacity(0),
+        //           ],
+        //         ),
+        //       ),
+        //       child: Text(
+        //         tr.alIqama,
+        //         style: theme.textTheme.displayMedium?.copyWith(
+        //             color: Colors.white,
+        //             fontWeight: FontWeight.w500,
+        //             shadows: kAfterAdhanTextShadow,
+        //             fontFamily:
+        //                 StringManager.getFontFamilyByString(tr.alIqama)),
+        //       ),
+        //     ).animate().slide(begin: Offset(0, -1)).fade().addRepaintBoundary(),
+        //   ),
+        // ),
```
please don't commit commented code

ismaelAbogabal commented:
@ibrahim-zehhaf-mawaqit
- removed commented code 
- fixed the bottom margins 

![image](https://github.com/mawaqit/android-tv-app/assets/63953869/ad1824ed-fab8-4a01-b2f4-f872b4353035)
